contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.1.0--3.2.0.sql

```diff
@@ -1,5 +1,5 @@
 -- complain if script is sourced in psql, rather than via ALTER EXTENSION
-\echo Use "ALTER EXTENSION ""babelfishpg_tsql"" UPDATE TO '3.1.0'" to load this file. \quit
+\echo Use "ALTER EXTENSION ""babelfishpg_tsql"" UPDATE TO '3.2.0'" to load this file. \quit
 
```

Comment:
We need to add the babelfish_drop_deprecated_object procedure as well. Example: https://github.com/babelfish-for-postgresql/babelfish_extensions/commit/aae61808f9f2c4447e95e6015f8c69b2ea835abd

Reply:
I didn't add file , just changed 1 ->2 in this file ( it seems it's wrong at the beginning ) ?